subiquity/models/filesystem.py

```diff
@@ -1763,7 +1788,13 @@ def needs_bootloader_partition(self):
                 "unknown bootloader type {}".format(self.bootloader))
 
     def _mount_for_path(self, path):
-        return self._one(type='mount', path=path)
+        mount = self._one(type='mount', path=path)
+        if mount is not None:
+            return mount
+        zpool = self._one(type='zpool', mountpoint=path)
+        if zpool is not None:
+            return zpool
+        return None
```

Comment:
minor: all these lines can be rewritten as:

```python
return self._one(type='zpool', mountpoint=path)
```

but I guess it can be intentional?